src/rational/q/from.rs

```diff
@@ -807,6 +825,41 @@ mod test_from_float {
     }
 }
 
+#[cfg(test)]
+mod test_into_float {
+    use super::*;
```

Comment:
I don't think that  a wildcard import is a problem here.

The reason not to use wildcards is that a change in the dependency could lead to name collisions when those are automatically imported after an update. This can result in code changing just by updating the dependencies.
This does not apply here since the parent module is not an external dependency.
More about this: https://www.lurklurk.org/effective-rust/wildcard.html